# HG changeset patch
# User Rik <rik@octave.org>
# Date 1443001554 25200
#      Wed Sep 23 02:45:54 2015 -0700
# Node ID cd4a8b4631da8b065a61256af94f6d11f4abd5c6
# Parent  df7d34a1c7e61566cee3a3ed91371e05e3d0fcc2
bitmax.m: Correct docstring to say function will be removed in version 4.6.

* bitmax.m: Correct docstring to say function will be removed in version 4.6.

diff --git a/scripts/deprecated/bitmax.m b/scripts/deprecated/bitmax.m
--- a/scripts/deprecated/bitmax.m
+++ b/scripts/deprecated/bitmax.m
@@ -14,17 +14,17 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {@var{r} =} bitmax (@var{precision})
 ##
-## @code{bitmax} is deprecated and will be removed in Octave version 4.4.
+## @code{bitmax} is deprecated and will be removed in Octave version 4.6.
 ## Use @code{flintmax (precision) - 1} for the equivalent functionality.
 ##
 ## Return the largest integer @var{r} that can be represented within a
 ## floating point value.
 ##
 ## The default class is @qcode{"double"}, but @qcode{"single"} is a valid
 ## option.  On IEEE 754 compatible systems, @code{bitmax} is
 ## @w{@math{2^{53} - 1}} for @qcode{"double"} and @w{@math{2^{24} - 1}} for
